packages/test-runner-webdriver/src/SessionManager.ts

```diff
@@ -84,6 +84,27 @@ export class SessionManager {
     return this.driver.keys(keys);
   }
 
+  async hover(_: string, selector: string) {
+    const { x, y } = await this.driver.$(selector).getLocation();
+    return this.driver.performActions([
+      {
+        type: 'pointer',
+        id: 'finger1',
+        parameters: {
+          pointerType: 'mouse',
+        },
+        actions: [
+          {
+            type: 'pointerMove',
+            duration: 0,
+            x: x + 1,
```

Comment:
Other runner _generally_ click/hover/etc in the middle of an element, does that matter?

Reply:
I agree it makes more sense to use something like [`middleOfNode`](https://github.com/vaadin/testing-helpers/blob/6e66938563adff75158374801b9af1948539f9ba/src/pointer.ts#L13) here.